src/components/forms/OrderErrorMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import { FaTriangleExclamation } from 'react-icons/fa6';
+
+export const OrderErrorMessage = () => {
+  return (
+    <div
+      className='border-error-medium rounded-lg border-2 p-4'
+      style={{ borderLeftWidth: '6px' }}
```

Comment:
Consider to replace inline styles with the tailiwnd classes, I add a plugigng so this could be spotted as warning in the future

Reply:
i will fix that tomorrow!

Reply:
I'll already merged, gonna fix it on my end. It's a tiny one